e2e-tests/cypress/tests/integration/channels/enterprise/ldap/ldap_login_spec.ts

```diff
@@ -220,17 +220,17 @@ function setLDAPTestSettings(config) {
 }
 
 function disableOnboardingTaskList(ldapLogin) {
-    cy.apiLogin(ldapLogin).then(({user}) => {
+    cy.apiLogin(ldapLogin).then((user) => {
         cy.apiSaveOnboardingTaskListPreference(user.id, 'onboarding_task_list_open', 'false');
         cy.apiSaveOnboardingTaskListPreference(user.id, 'onboarding_task_list_show', 'false');
         cy.apiSaveSkipStepsPreference(user.id, 'true');
     });
 }
 
 function removeUserFromAllTeams(testUser) {
-    cy.apiGetUsersByUsernames([testUser.username]).then(({users}) => {
+    cy.apiGetUsersByUsernames([testUser.username]).then((users) => {
         users.forEach((user) => {
```

Comment:
<img width="925" alt="image" src="https://github.com/mattermost/mattermost/assets/15979783/17538c56-8422-4338-bca6-698426679629">

Reply:
How can I get a license?, so that I'm able to run integration tests properly

Reply:
One option is to run the server with enterprise edition build via Docker, see https://github.com/mattermost/docker. Then once up and running, from there you may request for trial. Find something like "Try free for 30 days" or similar which is accessible using admin account (first user created).

Reply:
Thanks!